ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-cpou-upgrade-from-4.14.yaml

```diff
@@ -59,6 +59,12 @@ releases:
       architecture: arm64
       channel: fast
       version: "4.14"
+  source:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
```

Comment:
```suggestion
      version: "4.14"
```

Reply:
same to https://github.com/openshift/release/pull/57024/files#r1772434496